client-runtime/src/main/java/com/microsoft/rest/v2/http/HttpHeaders.java

```diff
@@ -62,6 +62,19 @@ public HttpHeaders add(String headerName, String headerValue) {
     }
 
     /**
+     * Replace the value for the header named headerName,
+     * discarding any value previously added for that header.
+     * @param headerName The name of the header.
+     * @param headerValue The value of the header.
+     * @return This HttpHeaders instance.
+     */
+    public HttpHeaders overwrite(String headerName, String headerValue) {
```

Comment:
I'd call it "set" instead of "overwrite". Overwrite is usually a flag that is passed to functions, not the name of a function.